src/iris/webhooks/rackspace.py

```diff
@@ -0,0 +1,94 @@
+from __future__ import absolute_import
+
+import datetime
+import logging
+import ujson
+from falcon import HTTP_201, HTTPBadRequest, HTTPNotFound
+
+from iris import db
+
+logger = logging.getLogger(__name__)
+
+
+class rackspace(object):
```

Comment:
This class has a lot of duplicated code with the alertmanager webhook. Could these inherit from a common webhook object?